application/build.gradle

```diff
@@ -55,8 +47,8 @@ dependencies {
   implementation 'org.flywaydb:flyway-core'
   implementation 'org.flywaydb:flyway-database-postgresql'
   implementation 'software.amazon.awssdk:cognitoidentityprovider'
-  implementation "de.siegmar:logback-awslogs-json-encoder:${logbackAwslogsJsonEncoderVersion}"
-  implementation "io.github.wimdeblauwe:error-handling-spring-boot-starter:${errorHandlingSpringBootStarterVersion}"
+  implementation "de.siegmar:logback-awslogs-json-encoder:2.0.0"
+  implementation "io.github.wimdeblauwe:error-handling-spring-boot-starter:4.5.0"
```

Comment:
I hardcoded them here as It seems dependabot can't update the versions when we outsource the version